### ISSUE TITLE
Group search showing archived ones

### ISSUE THREAD

**Issue by ghost** [BeWelcome/rox#228]
Feedback received from Polyglot :
"Group search is broken, returns all groups including archived."

**Comment by crumbking** [comment]
Do you mean:
https://www.bewelcome.org/groups/search?GroupsSearchInput=archive ?
I'm not sure if we have a flag to archive a group. Seems like they got just renamed.
Anyway really low priority.

**Comment by gbenouville** [comment]
Yes, it looks like the groups just get renamed with [archived]. But I guess they should't be diplayed in search results once they are archived.
Actually I don't really know what's the point of archiving, we could just delete them (it's not like anybody will ever go search a message in these old groups)

**Comment by thisismeonmounteverest** [comment]
The idea was that archiving them just by name would allow the group to be revived and the history of activity retained.
